contrib/scrooge/src/python/pants/contrib/scrooge/subsystems/scrooge_linter.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import multiprocessing
+from typing import Tuple
+
+from pants.option.option_util import flatten_shlexed_list
+from pants.subsystem.subsystem import Subsystem
+
+
+class ScroogeLinter(Subsystem):
+
+  options_scope = 'scrooge-linter'
+
+  @classmethod
+  def register_options(cls, register):
+    super().register_options(register)
+    register(
+      '--args', type=list, member_type=str, fingerprint=True,
+      help="Arguments to pass directly to the Scrooge Thrift linter, e.g. "
+           "`--scrooge-linter-args=\"--disable-rule Namespaces\"`",
```

Comment:
Is this a good example? I've never used the linter and don't know what it's `--help` looks like, beyond trying to parse the source code for Scrooge. I wasn't sure about capitalization.

The main point of these examples I've been adding to `args` is to show people that the strings are shlexable.

Reply:
I am also not sure! But I like the motivation. Thanks.